lib/parallel_tests/grouper.rb

```diff
@@ -129,6 +127,23 @@ def group_by_scenarios(tests, options = {})
         ParallelTests::Cucumber::Scenarios.all(tests, options)
       end
 
+      def separate_single_items(items, options)
+        items.partition { |item| to_single_items?(item, options) }
+      end
+
+      def to_single_items?(item, options)
```

Comment:
```suggestion
      def single_item?(item, options)
```

Reply:
this whole "item is item or item and size or item and tags" is kinda funky ... if you have any good ideas how to clean that up lmk or leave a TODO comment on how that can be improved